src/main/java/seedu/address/logic/commands/AssignmentIndivCommand.java

```diff
@@ -80,9 +80,9 @@ public CommandResult execute(Model model) throws CommandException {
         updatedAssignments.add(newAssignment);
 
         Person editedStudent = new Person(
-                studentToEdit.getName(), Optional.of(studentToEdit.getPhone()),
-                Optional.of(studentToEdit.getEmail()),
-                Optional.of(studentToEdit.getTelegramHandle()), Optional.of(studentToEdit.getAttendance()),
+                studentToEdit.getName(), Optional.ofNullable(studentToEdit.getPhone()),
+                Optional.ofNullable(studentToEdit.getEmail()),
+                Optional.ofNullable(studentToEdit.getTelegramHandle()), Optional.of(studentToEdit.getAttendance()),
```

Comment:
According to `.getAttendance()`, its implementation is
```java
return this.attendance.orElse(null);
```
This implies that it could be null, so would it be a better idea to use `Optional.ofNullable()` here too?